IRescue/UserLocalisation/Sensors/Marker/MarkerSensor.cs

```diff
@@ -145,18 +174,35 @@ public List<Measurement<Vector3>> GetPositions(long startTimeStamp, long endTime
         /// <returns>A list of all <see cref="Measurement{T}"/> and their time stamps and standard deviations.</returns>
         public List<Measurement<Vector3>> GetAllPositions()
         {
-            return this.positions;
+            return this.ArrayToList(this.positions);
+        }
+
+        /// <summary>
+        /// Convert an array to a list
+        /// </summary>
+        /// <param name="measurements">the array</param>
+        /// <returns>the list in the correct order of measurements</returns>
+        private List<Measurement<Vector3>> ArrayToList(Measurement<Vector3>[] measurements)
```

Comment:
The method sorts the array from New measurements to Old which cannot Be done with the ms one.